src/ui/views/main/components/menus/user_card.tsx

```diff
@@ -11,21 +19,95 @@ export interface IUserCardProps {
+    const { LL } = useI18nContext();
 
     let cached_user = useRootSelector((state: ReadRootState) => {
         let active_party = state.chat.active_party;
         return selectCachedUser(state, props.user.id, active_party);
     });
 
+    let user_discriminator = props.user.discriminator.toString(16).toUpperCase().padStart(4, '0');
+    
+    let state = useStructuredSelector({
+        is_light_theme: selectPrefsFlag(UserPreferenceFlags.LightMode),
+        party: (state: ReadRootState) => {
+            let party_id = activeParty(state);
+            if(party_id) {
+                return state.party.parties[party_id];
+            }
+            return;
+        },
+    });
+
+    
+    let member_info = createMemo(() => {
+        let members = state.party?.members;
+        for(let member of Object.values(members)) {
+            if(member.user.id == props.user.id) {
+                return member;
+            }
+        }
+        return
+    })
+    let background_color = createMemo(() => pickColorFromHash(props.user.id, state.is_light_theme));
+    let presence = createMemo(() => parse_presence(member_info()?.presence));
+
+    let status = createMemo(() => {
+        switch(presence().status) {
+            case PresenceStatus.Online: return [LL().main.ONLINE(), "online"];
+            case PresenceStatus.Busy: return [LL().main.BUSY(), "busy"];
+            case PresenceStatus.Away: return [LL().main.AWAY(), "away"];
+            default: return [LL().main.OFFLINE(), "offline"];
+        }
+    });
+
+    let nick = createMemo(() => member_info()?.nick);
+
+    
     return (
         <Show when={cached_user()} fallback={<span class="ui-text">User Not Found</span>}>
-            {cached_user => (
+            {cached_user => {
+                return (
+                    <>
                 <div class="ln-user-card ln-contextmenu">
-                    <span class="ui-text">User: {cached_user.user.username}</span>
-                </div>
-            )}
+                    <div class="banner" style={{"background-color":background_color()}}></div>
+                    <Avatar username={props.user.username} text={(nick() ?? cached_user.user.username)?.charAt(0)} backgroundColor={background_color()} rounded={true} />
+                    <div class="ln-user-status" title={status()[0]}>
+                        <Show
+                            fallback={<span class={status()[1]} />}
+                            when={presence().status == PresenceStatus.Online && presence().is_mobile}
+                        >
+                            <VectorIcon id={Icons.MobilePhone} />
+                        </Show>
+                    </div>
+                    <div class="ln-username" onClick={() => copyText(cached_user.user.username + '#'+user_discriminator)}>
+                        <Switch>
+                            <Match when={nick()}>
+                                <b class="font-large">{nick}</b>
```

Comment:
Instead of `font-large`, use the `ui-font` class + a header element, like `h4` or perhaps larger.